### ISSUE TITLE
Optician, optometrist, opthalmologist

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual-community-backlog#433]

**Comment by sarawilcox** [comment]
Proposal

### Optometrists, ophthalmologists and opticians
Optometrists check people's eyes for vision problems, or health issues such as high blood pressure. They offer advice, prescribe glasses and refer patients for treatment.

Ophthalmologists are doctors who specialise in eye and vision care. They are trained to perform eye surgery and can diagnose and treat eye diseases. They also prescribe medicine, glasses or contact lenses.

Opticians (sometimes called dispensing opticians) check prescriptions, fit and repair glasses and lenses, help with choosing frames and lenses.

**Comment by sarawilcox** [comment]
See the NHS website page on [Visiting an optician](https://www.nhs.uk/nhs-services/opticians/visiting-an-optician/) for more detail. 

Also Google trends. 
![Screenshot 2022-04-14 at 15 30 17](https://user-images.githubusercontent.com/45792959/163421108-897ecb47-034e-48c3-a6ee-1186adc0e7e5.png)
![Screenshot 2022-04-14 at 15 35 05](https://user-images.githubusercontent.com/45792959/163421137-d3bbd906-559f-46e1-b229-e595ac91bf6a.png)

**Comment by markasrobinson** [comment]
I have a slight concern that this could lead to "optician" being replaced with other more technical, less well known phrases and making advice less clear.

For example, "See an optician" could potentially be changed to "See an optometrist" based on this guidance, which - while technically correct - is not an action that will make sense for most users. People are not aware of what an optometrist is and how to go about seeing one.

Generally, we don't mention specific job titles too much because it's often not up to the user themselves to pick who they see. They access a service (e.g. GP, dentist, opticians) and are seen by/referred to an appropriate specialist (e.g. could actually be a practice nurse, hygienist, optometrist) - and they may never hear that person's job title mentioned at any point in their journey.

I think it might be worth flagging that guidance might be different when giving people actionable advice about accessing a service (generally an optician/opticians) vs describing the professionals a person might interact with.

**TL;DR** - we might want to consider promoting the point about saying "Go to an optician(s?)" when advising people about taking an action and only mention specific job roles when there is a clear reason to.

**Comment by sarawilcox** [comment]
### Reworded proposal for Style Council

Proposed text for style guide​:

We say "go to an optician" when we're telling people to take action. We only mention specific job roles where there's a user need for this. ​

Opticians (sometimes called dispensing opticians) check prescriptions, fit and repair glasses and lenses, and help with choosing frames and lenses.  We use "optician" for the place where you get your eyes tested. Do not say that an "optician" carries out eye tests. ​

Optometrists check people's eyes for vision problems, or health issues such as high blood pressure. They prescribe lenses and refer patients for treatment.

Ophthalmic medical practitioners are doctors who specialise in eye care. Like optometrists, they examine eyes, test vision, and prescribe lenses.​

Ophthalmologists are medical doctors who can perform eye surgery. They also diagnose and treat eye diseases, prescribe medicine, glasses or contact lenses.​

Orthoptists are specialists in squints and eye movement disorders.

**Comment by sarawilcox** [comment]
### April Style Council meeting

We discussed the entry at some length, in particular whether “optician” should be singular or plural. SEO, site search and names of individual businesses suggest we should use the plural. 

Proposed entry - APPROVED 

> We say “go to an opticians” (plural in this context) when we’re telling people to take action. We only mention specific job roles where there’s a user need for this.  
> 
> An optician (sometimes called a dispensing optician) checks prescriptions, fits and repairs glasses and lenses and helps with choosing frames and lenses. We use “opticians” for the place where you get your eyes tested. Note: an optician does not carry out eye tests.  
> 
> An optometrist carries out eye tests and checks for vision problems or health issues such as high blood pressure. They prescribe lenses and refer patients for treatment.  
> 
> An ophthalmic medical practitioner is a doctor who specialises in eye care. Like an optometrist, they examine eyes, test vision and prescribe lenses. 
> 
> An ophthalmologist is a medical doctor who can perform eye surgery. They also diagnose and treat eye diseases, and prescribe medicine, glasses or contact lenses.  
> 
> An orthoptist is a specialist in squints and eye movement disorders.  

Still needs clinical approval before publishing.

**Comment by sarawilcox** [comment]
Approved by clinicians.
